packages/webapp/pages/history.tsx

```diff
@@ -21,7 +21,7 @@ import { ButtonSize } from '@dailydotdev/shared/src/components/buttons/common';
 import { Dropdown } from '@dailydotdev/shared/src/components/fields/Dropdown';
 import { HistoryType, ReadingHistory } from '../components/history';
 import ProtectedPage from '../components/ProtectedPage';
-import { getLayout } from '../components/layouts/MainLayout';
+import { geHistoryLayout } from '../components/layouts/HistoryLayout';
```

Comment:
You probably took this from `SearchLayout` but feels a bit odd having the same component that doesn't introduce anything new.

I would maybe solve it the same way we do for `getDevCardLayout` or `getNotificationsLayout`
Just avoids having yet another wrapper.

(technically I think we should just remove the SearchLayout, but that's for another ticket.